templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes" class="btn btn-danger">

```

Comment:
add cancel button, it will be more user friendly in case you press delete button by mistake